packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -145,7 +166,10 @@ export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
 }
 
 /**
- * Implementation of the IFluidContainer.
+ * Base {@link IFluidContainer} implementation.
```

Comment:
@skylerjokiel Interested in your thoughts on how I documented this. Is there a reason this isn't an abstract class? It seems like it is intended to be consumed like one (unless I am misunderstanding something).

Reply:
It didn't use to be -- I must have missed when extending it became a thing.  In general we should probably push to unwind the concrete inheritance rather than going further down the rabbit hole with an abstract class, especially since one of the key design points of FluidContainer was to be service-agnostic.

Reply:
In that case, I'll go ahead and file a work item to clean this up, and update the docs to reflect the intended usage pattern (with a note about the current state)

Reply:
I filed [ADO 967](https://dev.azure.com/fluidframework/internal/_workitems/edit/967) to track the cleanup work. Please feel free to add any context as needed.